validation/validate.go

```diff
@@ -68,7 +68,11 @@ type SpecValidationError struct {
 }
 
 func (s StepValidationError) Error() string {
-	return fmt.Sprintf("%s:%d %s => '%s'%s", s.fileName, s.step.LineNo, s.message, s.step.GetLineText(), s.suggestion)
+	return fmt.Sprintf("%s:%d %s => '%s'", s.fileName, s.step.LineNo, s.message, s.step.GetLineText())
+}
+
+func (s StepValidationError) Suggestion() string {
+	return fmt.Sprintf("%s : %s => '%s'\t%s", "One or more errors were due to", "Step implementation not found", s.step.GetLineText(), s.suggestion)
```

Comment:
Agreed, It should show the `StepValidationError.message`.

Also it should check the `errorType` so that suggestion is valid for the given errortype.